integrations/react-ts/src/App.css

```diff
@@ -0,0 +1,86 @@
+.App {
```

Comment:
do we need a explanation here.  just some common css styles not necessary when doing integration